docs/src/main/tut/xor.md

```diff
@@ -7,3 +7,338 @@ scaladoc: "#cats.data.Xor"
+  final case class Right[+B](b: B) extends Xor[Nothing, B]
+}
+```
+
+Just like `Either`, it has two type parameters. Instances of `Xor` either hold a value
+of one type parameter, or the other. Why then does it exist at all?
+
+Taking a look at `Either`, we notice immediately there is no `flatMap` or `map` method on
+it. Instead, we must call `Either#right` (or `Either#left`) and then call `flatMap` or `map`
+on the result of that. The result of calling `Either#right` is a `RightProjection`, the name
+indicating what direction the `Either` is "biased" in. `RightProjection` is "right-biased", so
+calling `flatMap` or `map` on a `RightProjection` acts on the right type-parameter.
+
+```tut
+val e1: Either[String, Int] = Right(5)
+e1.right.map(_ + 1)
+
+val e2: Either[String, Int] = Left("hello")
+e2.right.map(_ + 1)
+```
+
+Note the return types are themselves back to `Either`, so if we want to make more calls to
+`flatMap` or `map` then we again must call `right` or `left`.
+
+More often than not we want to just bias towards one side and call it a day - by convention,
+the right side is most often chosen. This is the primary difference between `Xor` and `Either` -
+it is right-biased by default. `Xor` also has some more convenient methods on it, but the most
+crucial one is the right-biased being built-in.
+
+```tut
+import cats.data.Xor
+
+val xor1: Xor[String, Int] = Xor.right(5)
+xor1.map(_ + 1)
+
+val xor2: Xor[String, Int] = Xor.left("hello")
+xor2.map(_ + 1)
+```
+
+Because `Xor` is right-biased by default, it is easy to define a `Monad` instance for it (you
+could also define one for `Either` but due to how its encoded it may seem strange to fix a
+bias direction despite it intending to be flexible in that regard). Since we only ever want
+the computation to continue in the case of `Xor.Right` (as captured by the right-bias nature),
+we fix the left type parameter and leave the right one free.
+
+```tut
+import cats.Monad
+
+implicit def xorMonad[Err]: Monad[Xor[Err, ?]] =
+  new Monad[Xor[Err, ?]] {
+    def flatMap[A, B](fa: Xor[Err, A])(f: A => Xor[Err, B]): Xor[Err, B] =
+      fa.flatMap(f)
+
+    def pure[A](x: A): Xor[Err, A] = Xor.right(x)
+  }
+```
+
+### Example usage: Round 1
+As a running example, we will have a series of functions that will parse a string into an integer,
+take the reciprocal, and then turn the reciprocal into a string.
+
+In exception-throwing code, we would have something like this:
+
+```tut
+object Exceptiony {
+  def parse(s: String): Int =
+    if (s.matches("-?[0-9]+")) s.toInt
+    else throw new NumberFormatException(s"${s} is not a valid integer.")
+
+  def reciprocal(i: Int): Double =
+    if (i == 0) throw new IllegalArgumentException("Cannot take reciprocal of 0.")
+    else 1.0 / i
+
+  def stringify(d: Double): String = d.toString
+}
+```
+
+Instead, let's make the fact that some of our functions can fail explicit in the return type.
+
+```tut
+object Xory {
```

Comment:
Really nit-picky suggestion, use `ExceptionStyle` and `XorStyle` instead. I worry that the `Xory` name might not obviously read as you intend, i.e. "xor-y" versus "exception-y"